queue-scm-1.rockspec

```diff
@@ -27,7 +27,8 @@ build = {
         ['queue.abstract.driver.limfifottl'] = 'queue/abstract/driver/limfifottl.lua',
```

Comment:
Let's also start with CI check draft here, this things must be pushed together.

Reply:
Implemented job `hardcoded_ver_check` in `fast_testing.yml`